org.gnome_terminator.Terminator2.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you should patch the upstream file. submit the patch upstream and add the patch instead of the full file. so that next release you can remove the patch.